stdlib/public/core/MigrationSupport.swift

```diff
@@ -2,16 +2,23 @@
 //
 // This source file is part of the Swift.org open source project
 //
-// Copyright (c) 2014 - 2018 Apple Inc. and the Swift project authors
+// Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
 // Licensed under Apache License v2.0 with Runtime Library Exception
 //
 // See https://swift.org/LICENSE.txt for license information
 // See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 //===----------------------------------------------------------------------===//
-
+//
 // This file contains only support for types deprecated from previous versions
-// of Swift
+// of Swift.
+//
+// Placeholders:
+//
+// * `deprecated: 100000` is API_TO_BE_DEPRECATED in the next version of Swift.
+// * `obsoleted: TODO` is for a future (source-breaking) language version mode.
+//
+//===----------------------------------------------------------------------===//
```

Comment:
This is for "soft deprecation" (until we know the next Swift version number).

See also: [swift-corelibs-foundation](https://github.com/apple/swift-corelibs-foundation/blob/514c9e9ea90bbb6596bfbf343db205d4d588cc56/Sources/Foundation/FileHandle.swift#L646-L694)